### PR TITLE
Fills out settings page and additional updates

### PR DESCRIPTION
**Pull request by sterlingwalker**
## Overview

- Adds Privacy Policy, TOS, and Cookie policy to settings page
- Adds Feedback and Contact screen to settings page
- Adds controls to update views, activity, and status

### CODE REVIEW

**Comment by cheesea3** [comment]
Awesome stuff, Sterling! I love all that you brought into the UI. I love the creative freestyle approach to bringing these features to life! You brought some unique ideas with it. I added some comments which are just minor type enforcements along with removing unnecessary imports. If you are ready to merge then I go ahead! Once you're ready, I'll add a commit and put up another pr for you to review as I built some features on top of your awesome work.

LGTM 💯

**Review by cheesea3 — APPROVED** [review]
Awesome stuff, Sterling! I love all that you brought into the UI. I love the creative freestyle approach to bringing these features to life! You brought some unique ideas with it. I added some comments which are just minor type enforcements along with removing unnecessary imports. If you are ready to merge then I go ahead! Once you're ready, I'll add a commit and put up another pr for you to review as I built some features on top of your awesome work.

LGTM 💯

File: src/components/StatusChange.tsx [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+  const [tabIndex, setTabIndex] = useState(0)
+  const handleTabChange = (value: number) => {
+    setTabIndex(value)
+  }
+
+  const handleChangeIndex = (index: number) => {
+    setTabIndex(index)
+  }
+
+  const printSheetDark = () => {
```
[the remark this answers is not in the document]
Comment (sterlingwalker) [reply src/components/StatusChange.tsx]: Great suggestion! This naming  convention makes it a lot more clear that the styling isn't only one sided. I'm also still getting used to implementing the light/dark mode styling so I really appreciate the help!

File: src/components/StatusChange.tsx [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+                          color={viewIndex == 2 ? 'light' : 'bg-zinc-900'}
+                          className='mr-2'
+                          icon={checkmarkOutline}
+                        ></IonIcon>
+                        Expanded
+                      </a>
+                    </div>
+                    <div>
+                      <span className='text-gray-100 text-xs'>
+                        Each post is shown in it's entirety
```
[the remark this answers is not in the document]
Comment (sterlingwalker) [reply src/components/StatusChange.tsx]: I actually didn't know this was a thing, very interesting to know, thanks!

File: src/components/PolicyContainer/PolicyContainer.tsx [hunk 3]
```diff
@@ -11,18 +11,11 @@ import {
...
-  setShowPolicy(showPolicy: boolean): boolean
-  showPolicy: boolean
-  Sections: Sections[]
-  Title: string
-  Subtitle: string
+  heading: String
+  description: String
 }
 
 function PolicyContainer({
```
[the remark this answers is not in the document]
Comment (sterlingwalker) [reply src/components/PolicyContainer/PolicyContainer.tsx:25]: Thanks for handling, I fully agree that this is the proper way of doing this!

**Pull request merged**